src/Dfc.CourseDirectory.Web/ViewComponents/Courses/Duration/Duration.cs

```diff
@@ -3,13 +3,28 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using Dfc.CourseDirectory.Models.Models.Courses;
+using Dfc.CourseDirectory.Web.Helpers;
+using Microsoft.AspNetCore.Mvc.Rendering;
 
 namespace Dfc.CourseDirectory.Web.ViewComponents.Courses.Duration
 {
     public class Duration : ViewComponent
     {
         public IViewComponentResult Invoke(DurationModel model)
         {
+            model.DurationUnits = new List<SelectListItem>();
+            foreach (DurationUnit eVal in DurationUnit.GetValues(typeof(DurationUnit)))
+            {
+                if (eVal.ToString().ToUpper() != "UNDEFINED")
+                {
+                    var item = new SelectListItem { Text = WebHelper.GetEnumDescription(eVal) };
+                    model.DurationUnits.Add(item);
+                }
+            };
+
+            model.DurationUnits.Insert(0,model.DurationUnits.FirstOrDefault(x => x.Text.ToUpper()=="HOURS"));
```

Comment:
If we had hours as 1, we would not need to do these lines of code.
And we have to repeat it every time we have DurationUnit dropdown.